### PR TITLE
Keycloak DevDay 2024 videos

### PR DESCRIPTION
**Pull request by dasniko**
@ahus1 and @srose please review.

### CODE REVIEW

File: blog/2024/keycloak-dev-day-videos.adoc [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+* https://youtu.be/5WjIEj9HLk8?si=HdBLw2PNwyTCqqnb[A Look at Keycloak from the IAM Point of View (Robert Bauer, intension)]
+* https://youtu.be/Xqtd8EUxakk?si=IEcXZpYnLx_Y9Eby[Automating Keycloak Configuration (Sophie Tauchert, Relaxdays)]
+* https://youtu.be/mgarRWSQLUc?si=gemXaXJGfDsWD31q[Ask the Maintainers Session (A. Schwartz, T. Darimont, S. Schuster)]
+* https://youtu.be/w2T-NmnOaTE?si=K84dI5yu7n-uMHRm[IAM Doomsday Prepper (Maik Kingma)]
+* https://youtu.be/mGOrlC6ywNI?si=EaLFTo4ss4EYEq6r[Keycloak Cassandra Datastore (Dominik Schlosser)]
+
+The complete
+playlist can be found here: https://www.youtube.com/playlist?list=PLNn3plN7ZiaqXjiDSB1KDaF3bCDuQmDgw
+
+We are currently having the first preparations for the next edition of the DeveloperDay in 2025.
```
Comment (srose) [inline blog/2024/keycloak-dev-day-videos.adoc:27]: Proposal: 
We are currently preparing the next edition "Keycloak DeveloperDay 2025".
If you want to contribute, please get in touch with us (Sebastian & Niko)!

Comment (dasniko) [reply blog/2024/keycloak-dev-day-videos.adoc:27]: Thanks, Sebastian.
Should I adjust only this one sentence or replace the complete last paragraph?

**Review by ahus1 — APPROVED** [review]
Thank you for this post in our blog! I added one commit with just little changes.

**Comment by ahus1** [comment]
The blog post will be live within some minutes as the main Keycloak site build is complete which takes a while due to the size of the main site which also include all the docs. You can track the progress here: https://github.com/keycloak/keycloak.github.io/actions

Feel free to announce the blog post once it is online on the keycloak user mailing list.

**Pull request merged**